Cargo.toml

```diff
@@ -45,6 +45,9 @@ boa_profiler = { version = "~0.20.0", path = "core/profiler" }
 boa_runtime = { version = "~0.20.0", path = "core/runtime" }
 boa_string = { version = "~0.20.0", path = "core/string" }
 
+# Utility Repo Crates
+tag_ptr = { path = "utils/tag_ptr" }
```

Comment:
Can you test if this doesn't require the version?
```
cargo ws publish --dry-run
```
I remember that the dependency still required the version when publishing to crates.io, but maybe things changed.

Reply:
```bash
cargo ws publish --dry-run
```

Seems to work :)

```txt
info checking tag_ptr
   Compiling tag_ptr v0.1.0 (/home/arch/projects/boa/utils/tag_ptr)
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.10s
    Updating crates.io index
   Packaging tag_ptr v0.1.0 (/home/arch/projects/boa/utils/tag_ptr)
    Packaged 7 files, 14.6KiB (5.6KiB compressed)
   Uploading tag_ptr v0.1.0 (/home/arch/projects/boa/utils/tag_ptr)
warning: aborting upload due to dry run
```